lunes_cms/static/js/color_unread_feedback.js

```diff
@@ -0,0 +1,19 @@
+/*
+ * Function to highlight unread feedback with bold letters
+*/
+
+
+function markUnreadFeedback () {
+    let readByEntries = document.querySelectorAll(".field-read_by");
+    if (readByEntries.length != 0) {
```

Comment:
```suggestion
    if (readByEntries.length > 0) {
```
Just to be safe and eliminated the theoretical option to insert -1 as length